templates/hackmd/0/docker-compose.yml

```diff
@@ -0,0 +1,43 @@
+hackmdPostgres:
+  image: postgres
+  environment:
+    POSTGRES_USER: hackmd
+    POSTGRES_PASSWORD: hackmdpass
+    POSTGRES_DB: hackmd
+
+hackmd:
+  image: hackmdio/hackmd:0.4.5
+  environment:
+    POSTGRES_USER: hackmd
+    POSTGRES_PASSWORD: hackmdpass
+    NODE_ENV: "${NODE_ENV}"
+    DEBUG: "${DEBUG}"
+    HMD_DOMAIN: "${HMD_DOMAIN}"
+    HMD_URL_PATH: "${HMD_URL_PATH}"
+    HMD_PORT: "${HMD_PORT}"
```

Comment:
As I tould you previously in other MR, I actually have the rancher load-balancer enabled and that's why I don't like any rancher app to have ports open, because there's no need to. This way you just need to point to that container with rancher's lb and set which internal port it works on.

I needed to edit each template I've used from this repo because most of them are node apps working at port 3000, making them all collide.

If you want I'll do like in redmine stack, where I've not opened the port but I've added a readme file specifying it; but if I can prevent adding that rule, I would.